packages/addon/lua/gelly/api/mods/manager.lua

```diff
@@ -1,86 +1,103 @@
 gellyx = gellyx or {}
 gellyx.mods = gellyx.mods or {}
-gellyx.mods.loadedMods = gellyx.mods.loadedMods or {}
 
-local INVALID_PARSE = { Valid = false }
-
-local CRUCIAL_FILES = {
-	"info.lua",
-	"init.lua",
-}
-
-local function parseMod(path)
-	for _, crucialFile in ipairs(CRUCIAL_FILES) do
-		if not file.Exists(path .. "/" .. crucialFile, "LUA") then
-			return INVALID_PARSE
-		end
-	end
+local repository = include("gelly/api/mods/mod-repository.lua")
+local findAllGellyMods = include("gelly/api/mods/find-all-gelly-mods.lua")
+local array = include("gelly/util/functional-arrays.lua")
+local logging = include("gelly/logging.lua")
+
+local loadedMods = {}
+
+local DEFAULT_MOD = "blood-mod"
+
+include("gelly/api/mods/enums.lua")
+include("gelly/api/mods/restrict-mod-additions.lua")
```

Comment:
Why is this included as a side effect? If anyone happens to re-order the code where this isn't called during initialization, then it'll break. I'd suggest changing this to return the function, it's also not even used by any other module